addons/website/static/tests/tours/page_manager.js

```diff
@@ -109,3 +109,40 @@ wTourUtils.registerWebsitePreviewTour('website_page_manager', {
         run: () => null,
     },
 ]);
+
+wTourUtils.registerWebsitePreviewTour('website_page_manager_session_forced', {
+    test: true,
+    url: '/',
+}, [{
+    content: "Open the website switcher",
+    trigger: '.o_website_switcher_container button',
+}, {
+    content: "Switch to website 2",
+    // Ensure data-website-id exists
+    extra_trigger: 'iframe html[data-website-id="1"]',
+    trigger: '.o_website_switcher_container .dropdown-item:contains("My Website 2")'
+}, {
+    content: "Wait for the iframe to be loaded",
+    // The page reload generates assets for website 2, it may take some time
+    timeout: 20000,
```

Comment:
Really? Are not all assets supposed to be generated already when starting the test?

Reply:
This part is just a copy paste of [another test doing the same thing](https://github.com/odoo/odoo/commit/d7b2b7882fa7ea6d7f50bcdd59a51674ddd0fb15#diff-c94b7a6e57ad4103671929d1c80a5c5e7c4b76d9c8ac1819aeaf9688ec6092eaR17-R31).
And now that you mention it, I would think the same but it seems like actually, no, see https://github.com/odoo/odoo/commit/40367e36837a157a0422462487960cf4c54601e5

Since it's not a new line but a copy paste, I'd rather not change it

Reply:
Ok :shrug: